appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/ModuleDescriptor.java

```diff
@@ -40,9 +40,12 @@
 package fish.payara.micro.data;
```

Comment:
These commits are form a different change

Reply:
They were from PAYARA-2300, since PAYARA-2301 was branched off of that one. This split was done to make the PRs more manageable. But with PAYARA-2300 now merged they should have automatically disappeared from the diff.